### PR TITLE
MES-2048 - show eta on office page

### PR DESCRIPTION
**Pull request by roberthall12**
## Description and relevant Jira numbers

Added ETA fault display to office page, 
https://jira.i-env.net/browse/MES-2048
Also added ECO faults display (MES-2049) while waiting on initial office form creation,
https://jira.i-env.net/browse/MES-2049

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [x] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval
Screen Shot 2019-04-11 at 16 47 46

### CODE REVIEW

**Comment by ammarhaiderbjss** [comment]
@roberthall12 quick screenshot?

**Comment by roberthall12** [comment]
> @roberthall12 quick screenshot?

**Comment by roberthall12** [comment]
Added screenshot

File: src/pages/office/office.html [hunk 1]
```diff
@@ -19,4 +19,36 @@
         <button ion-button navPush="TerminateTestPage">Terminate Test</button>
       </ion-card-content>
     </ion-card>
+
+    <ion-card id="ETA" *ngIf="(pageState.etaFaults$ | async)">
+        <ion-card-content>
+          <ion-grid>
+            <ion-row align-items-start class="mes-data">
+              <ion-col col-3></ion-col>
```
Comment (Cavalier07) [inline src/pages/office/office.html]: Why is there an Empty ion-col?

File: src/pages/office/__tests__/office.spec.ts [hunk 2]
```diff
@@ -42,6 +75,27 @@ describe('OfficePage', () => {
     });
   });
 
+  describe('DOM', () => {
+    it('should hide ETA faults container if there are none', () => {
+      fixture.detectChanges();
+      expect(fixture.debugElement.query(By.css('#ETA'))).toBeNull();
+    });
+    it('should display ETA faults container if there are any', () => {
```
Comment (Cavalier07) [inline src/pages/office/__tests__/office.spec.ts]: Is it worth testing the Physical ETA here or are we sure it works for both based on tests elsewhere

File: src/pages/office/office.html (hunk 1 above)
Comment (roberthall12) [reply src/pages/office/office.html]: its to do with the spacing between the label and the fault, is there another way to space it without adding the column?

Comment (buzzub) [reply src/pages/office/office.html]: Just add left padding to the `.fault-heading` class?

File: src/pages/office/office.html [hunk 3]
```diff
@@ -19,4 +19,36 @@
...
+              <ion-col col-34>
+                <h4 class="fault-heading">ETA</h4>
+              </ion-col>
+              <ion-col align-self-start>
+                <div id="etaFaults">{{pageState.etaFaults$ | async}}</div>
```
[the remark this answers is not in the document]
Comment (roberthall12) [reply src/pages/office/office.html]: added

File: src/pages/office/office.html (hunk 1 above)
Comment (roberthall12) [reply src/pages/office/office.html]: fixed

**Pull request merged**